pkgs/http_profile/pubspec.yaml

```
@@ -0,0 +1,15 @@
+name: http_profile
+description: >-
+  A library used by HTTP client authors to integrate with the DevTools
+  Network tab.
+publish_to: none
+repository: https://github.com/dart-lang/http/tree/master/pkgs/http_profile
+
+environment:
+  sdk: '>=3.0.0 <4.0.0'
```

[nit] With Dart 3 we hav the option to use caret constraints for the SDK.

```suggestion
  sdk: ^3.0.0
```